pages/home.js

```diff
@@ -181,80 +199,89 @@ export default function Home(props) {
             pathname: `/room/${addRoomRes.data.addRoom.id}`,
           })
           .catch((e) => {
-            throw 'Oops! Something went wrong'
+            throw t.genericError
           })
       } else {
-        throw 'Oops! Something went wrong'
+        throw t.genericError
       }
     } catch (e) {
       setCreateRoomError(e)
     }
   }
   return (
-    <div
-      data-testid="homeContent"
-      id="homeContent"
-      className="container grid grid-cols-1 gap-y-5 mx-auto sm:flex sm:justify-center sm:gap-x-5"
-    >
-      <Container style="text-center p-4 flex flex-col drop-shadow md:w-96">
-        <h2 className="text-opacity-75 text-black font-bold text-2xl">
-          {t.createRoomTitle}
-        </h2>
-        <h3 className="text-opacity-75 text-black text-xl">
-          {t.createRoomDesc}
-        </h3>
-        <form
-          data-testid="createRoomForm"
-          onSubmit={onCreateHandler}
-          className="flex flex-col justify-between h-full items-center"
-        >
-          {createRoomError ? (
-            <ErrorLabel message={createRoomError}></ErrorLabel>
-          ) : undefined}
-          <TextInput
-            id="owner"
-            label={t.createRoomLabel}
-            placeholder={t.createRoomPlaceholder}
-          />
-          <button
-            type="submit"
-            className="w-max font-display text-white bg-[#318000] hover:bg-[#1D4D00] active:bg-[#102900] py-3 px-5 rounded mt-12 focus:drop-shadow focus:ring-2 focus:ring-gray-600 border border-[#458259] text-[22px] leading-8 [text-shadow:1px_2px_0px_#333]"
+    <>
+      {queryErrorCode ? (
+        <ErrorLabel message={errorCodeMsg}></ErrorLabel>
+      ) : undefined}
```

Comment:
Great job! Mostly nitpicking as I think this looks pretty good so I'll approve but I think the error label could use a bit of bottom padding. Something like this:

```suggestion
      {queryErrorCode ? (
      <div className="pb-4">
          <ErrorLabel message={errorCodeMsg}></ErrorLabel>
       </div>
      ) : undefined}
```